iree/base/status_test.cc

```diff
@@ -26,15 +26,11 @@ using ::testing::HasSubstr;
 #if (IREE_STATUS_FEATURES & IREE_STATUS_FEATURE_ANNOTATIONS) != 0
```

Comment:
Shouldn't this be `&&`?

Reply:
Nope. It's a bitfield, so test for features using bitwise operators.

Reply:
Aaaah